MiniApp.xcodeproj/project.pbxproj

```diff
@@ -31,8 +31,10 @@
 		38F3B7F024192E7300914E37 /* UrlParserTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 38F3B7EF24192E7300914E37 /* UrlParserTests.swift */; };
 		38F73B4523D554FE00F2C1CE /* Helpers.swift in Sources */ = {isa = PBXBuildFile; fileRef = 38F73B4423D554FE00F2C1CE /* Helpers.swift */; };
 		38F73B4723D55CCB00F2C1CE /* MiniAppInfoFetcherTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 38F73B4623D55CCB00F2C1CE /* MiniAppInfoFetcherTests.swift */; };
+		58A0DF3B3769B8951169516B /* Pods_MiniApp_Tests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 52C30D2FD93FB4105B5C47EA /* Pods_MiniApp_Tests.framework */; };
```

Comment:
Please avoid pushing this change into GIT, you can do this by pod deintegrate. If these changes are pushed, then for every PR, we have to review this file.